jwst/regtest/test_NIRSpec_IRS2_detector1.py

```diff
@@ -0,0 +1,86 @@
+""" Test for the detector1 pipeline using NIRSpec data in IRS2 mode. This takes
+    an uncal file and generates the stage 1 FITS files (rate) along with the
+    intermediate products."""
+
+import os
+from glob import glob
+import pytest
+
+from astropy.io.fits.diff import FITSDiff
+from jwst.stpipe import Step
+
+@pytest.fixture(scope="module")
+def run_pipeline(rtdata_module, jail):
+    """Run calwebb_Detector1 pipeline on NIRSpec uncal data. This test is for
+       data taken with the IRS2 mode. The steps are similar to the NIRCam level
+       2a test but the IRS2 mode tests different paths in the same modules."""
+    rtdata = rtdata_module
+    rtdata.get_data("nirspec/test_detector1Pipeline/jw0010010_11010_nrs1_chimera_uncal.fits")
+
+    Step.from_cmdline(["jwst.pipeline.Detector1Pipeline", rtdata.input,
+                       "--save_calibrated_ramp=True",
```

Comment:
You don't need to save the calibrated ramp if you save the output from the `jump` step, because they would be the same.